### DISCUSSION TITLE
How many 🍌🍌 did I start with?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#597] [Galaxy Trivia] [answered]
After a day harvesting 🍌🍌 I had a basket full.

I ate 5 and gave Alex half of the remaining.

I then ate another 3 and gave Billie one third of the remaining.

I ate another 6 and then gave Charlie two thirds of the remaining.

I now had 34 left.

**Comment by peterbaccaro** [comment, marked as answer]
**335 bananas**.

Working backwards...

I ended up with 34 bananas, so I had 102 before giving Charlie two thirds.
I ate 6 myself, so I had 108 before I ate those.
I had 108 bananas, so I had 162 before giving Billie one third.
I ate 3 myself, so I had 165 before I ate those.
I had 165 bananas, so I had 330 before giving Alex one half.
Ate 5 myself, so I had 335 before I ate those.
Therefore I started with 335 bananas.

**Comment by thuongtruong1009** [comment]
Let's work backwards from the end result of 34 bananas to figure out how many bananas you started with:

After giving Charlie two thirds of the remaining bananas, you had 34 left. This means that you had 3 times the remaining bananas before giving them to Charlie. So, 3 * 34 = 102.
Before giving the bananas to Charlie, you ate 6, leaving you with 2/3 of the remaining bananas. This means that you had 3 times the remaining bananas before eating the 6. So, 3 * (34 + 6) = 120.
Before giving the bananas to Billie, you had 1/3 of the remaining bananas. This means that you had 3 times the remaining bananas before giving them to Billie. So, 3 * (34 + 3) = 111.
Before giving the bananas to Alex, you had half of the remaining bananas. This means that you had twice the remaining bananas before giving them to Alex. So, 2 * (111 + 5) = 232.
Therefore, you started with 232 bananas.
